src/lib/tests/e2e/dashboard.e2e.test.js

```diff
@@ -46,3 +48,67 @@ describe('dashboard e2e', () => {
     server.close();
   });
 });
+
+
+describe('port, config, appId, serverURL, masterKey, appName, graphQLServerURL options should not be ignored.', () => {
+  it('Should start with port 4041.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--port', '4041']);
+
+    expect(result).toContain('The dashboard is now available at http://0.0.0.0:4041/');
+  });
+
+  it('Should return an error message if config and appId options are provided together.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', '--appId', 'helloworld']);
+
+    expect(result).toContain('You must provide either a config file or other CLI options (appName, appId, masterKey, serverURL, and graphQLServerURL); not both.');
+  });
+
+  it('Should return an error message if config and serverURL options are provided together.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', '--serverURL', 'helloworld']);
+
+    expect(result).toContain('You must provide either a config file or other CLI options (appName, appId, masterKey, serverURL, and graphQLServerURL); not both.');
+  });
+
+  it('Should return an error message if config and masterKey options are provided together.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', '--masterKey', 'helloworld']);
+
+    expect(result).toContain('You must provide either a config file or other CLI options (appName, appId, masterKey, serverURL, and graphQLServerURL); not both.');
+  });
+
+  it('Should return an error message if config and appName options are provided together.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', '--appName', 'helloworld']);
+
+    expect(result).toContain('You must provide either a config file or other CLI options (appName, appId, masterKey, serverURL, and graphQLServerURL); not both.');
+  });
+
+  it('Should return an error message if config and graphQLServerURL options are provided together.', async () => {
+    const result = await startParseDashboardAndGetOutput(['--config', 'helloworld', '--graphQLServerURL', 'helloworld']);
+
+    expect(result).toContain('You must provide either a config file or other CLI options (appName, appId, masterKey, serverURL, and graphQLServerURL); not both.');
+  });
+});
+
+function startParseDashboardAndGetOutput(args) {
+  const TIMEOUT = 1000; // ms
```

Comment:
Rename to `timeoutInMs` and remove comment, we use camelcase for vars.

Reply:
Done